appserver/javaee-api/javax.javaee-api/pom.xml

```diff
@@ -125,13 +125,13 @@
             <optional>true</optional>
         </dependency>
         <!--
-            using com.sun.mail.java.mail instead of javax.mail:javax.mail-api
-            because javax.mail has a dependency to com.sun.mail classes,
+            using com.sun.mail.java.mail instead of jakarta.mail:jakarta.mail-api
+            because jakarta.mail has a dependency to com.sun.mail classes,
```

Comment:
Is this comment yet true (after the replace?)

Reply:
yes